src/services/misc/lazerpay.getAcceptedCoins.ts

```diff
@@ -13,7 +13,7 @@ export default async function(args: TransactionData) {
     const response = await LazerApi.get(API_URL_GET_ACCEPTED_COINS);
 
     return response?.data;
-  } catch (err) {
-    return err;
+  } catch (err: any) {
+    throw err?.response?.data;
```

Comment:
Hi @Lord-Jerry, Instead of throwing .resonse?.data and the optional chain, why not just throw the full error ?
```js
throw error;
```

Reply:
@iamnotstatic it includes unnecessary request objects.


Didn't think they we're needed